### PR TITLE
LPD-32219 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://liferay.atlassian.net/browse/LPD-32219

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
Blocked by: https://liferay.atlassian.net/browse/LRCI-4416

More details [here](https://liferay.slack.com/archives/C06A59K5D46/p1722357953135419)

File: modules/test/playwright/tests/analytics-settings-web/connectionInformation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test(
+	'Modify the sites and channels synchronized in the property review sidebar',
+	{
+		tag: '@Legacy',
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/analytics-settings-web/connectionInformation.spec.ts:29]: Hey @marcosapmf, the correct tag should be `LRAC-11044`. You can see this information at the beginning of the `@description` tag in the Poshi test code ([Here](https://github.com/liferay-ac/liferay-portal/pull/1717/files#diff-ef61baf25231b1dcc39ad9212b3c128731413ebd0fbbeb79f4003d72882a1faeL212))

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
Covered by: https://github.com/liferay-ac/liferay-portal/pull/1718

**Pull request closed without merging**